receiver/awscontainerinsightreceiver/internal/stores/podstore.go

```diff
@@ -472,13 +472,19 @@ func (p *PodStore) addStatus(metric CIMetric, pod *corev1.Pod) {
 					switch {
 					case containerStatus.State.Running != nil:
 						metric.AddTag(ci.ContainerStatus, "Running")
+						metric.AddField(ci.ContainerStatusRunning, 1)
```

Comment:
Once you move the consts above, the key here will be derived as `ci.MetricName(<type>, <measurement>)`

Reply:
Yup! I originally thought this was the root issue but realized it's just string concatenation